ironfish-cli/src/commands/wallet/transaction/watch.ts

```diff
@@ -39,12 +40,22 @@ export class WatchTxCommand extends IronfishCommand {
 
     const client = await this.sdk.connectRpc()
 
-    await watchTransaction({
-      client,
-      logger: this.logger,
-      account,
-      hash,
-      confirmations: flags.confirmations,
-    })
+    const timeout = 5000
+    let watched = false
+
+    while (!watched) {
+      watched = await watchTransaction({
```

Comment:
`watchTransaction` is a CLI utility function -- if we're just looping until the transaction is found, could we do the loop inside of `watchTransaction`?

Reply:
moved the loop to `watchTransaction`